### DISCUSSION TITLE
Latest version of highcharts-more.js is giving out errors in console on tooltips

### DISCUSSION THREAD

**Discussion by James-76** [poblabs/weewx-belchertown#796] [General]
Hello

It appears that they have released version V10.2 ([http://code.highcharts.com/10.2/highcharts-more.js](http://code.highcharts.com/highcharts-more.js)) a couple of days ago and since then tooltips on circular graphs are giving out errors and not working as designed.

![image](https://user-images.githubusercontent.com/59254774/177733080-5fe501ae-de3a-40ce-afb0-948020379d45.png)

In summary it can be fixed editing header.html.tmpl and forcing it to use the older version of the code.   My skills are not up to working out if this is a bug or an intended change within the highcharts code, therefore not submitting a pull request for this.

old:
`        <script type='text/javascript' src='//code.highcharts.com/highcharts-more.js'></script>`

new:
`        <script type='text/javascript' src='//code.highcharts.com/10.1/highcharts-more.js'></script>`

James

**Comment by James-76** [comment]
I have worked out it has been reported to Highcharts at https://github.com/highcharts/highcharts/issues/17472.

**Reply by poblabs** [reply]
Should we specify a specific version of highcharts?

**Reply by James-76** [reply]
It is an interesting topic, and there are pros and cons here as you know

In some ways it is a good idea because you control which code level you are running at, however you do then have to periodically go through and review in case there is newer features or security fixes.

At this point I'm wondering if we see how long it is before the issue is fixed as the code was only released a couple of days ago.   If they are looking like they are taking ages to fix we may wish to consider fixing to a release level.

James

**Reply by CaptainSteubing** [reply]
I'd say perhaps fix to a specific version or make it an option. I used the solution proposed by @James-76 and modified header.html.tmpl and the Hays Chart tooltips are working again.

Thanks!

**Comment by norecords** [comment]
I've build my own highcharts files from the master repo that fix lot's of issues.
You can download them here and use them until next Highcharts release
https://meteo.correns.org/highcharts/10.2.0-commit-17634/highstock.js
https://meteo.correns.org/highcharts/10.2.0-commit-17634/highcharts-more.js
https://meteo.correns.org/highcharts/10.2.0-commit-17634/modules/exporting.js
https://meteo.correns.org/highcharts/10.2.0-commit-17634/modules/solid-gauge.js
That's also solved #812

**Comment by James-76** [comment]
This was fixed under 10.2.1 which was released as scheduled at the end of August

```
/*
 Highcharts JS v10.2.1 (2022-08-29)

 (c) 2009-2021 Torstein Honsi

 License: www.highcharts.com/license
*/
```
